imgs/default.nix

```diff
@@ -1,6 +1,8 @@
 { pkgs }:
 
 rec {
+  awsutils = pkgs.callPackage ./ansible-lint { };
```

Comment:
youll need to update the name to `ansible-lint` here as well :)

Reply:
:man_facepalming: